### PR TITLE
Update easidao.toml

### PR DESCRIPTION
**Pull request by hypn0t1z**

### CODE REVIEW

File: data/ecosystems/e/easidao.toml [hunk 1]
```diff
@@ -3,59 +3,17 @@ title = "EasiDAO"
...
-url = "https://github.com/EaseDeFi/snapshot-spaces"
-
-[[repo]]
-url = "https://github.com/EaseDeFi/synthetix-faq-bot"
-
-[[repo]]
-url = "https://github.com/EaseDeFi/yield-server"
-
-[[repo]]
-url = "https://github.com/EaseDeFi/yinsure-finance"
```
Comment (eherrerosj) [inline data/ecosystems/e/easidao.toml:61]: why are you proposing to remove this repos?

**Review by eherrerosj — CHANGES_REQUESTED** [review]
we usually keep all repos including 404s, public repos that became private, forks, etc because we have ways to handle them

**Review by jubos — CHANGES_REQUESTED** [review]
Please don't remove the existing repos as we keep them for archival support.

File: data/ecosystems/e/easidao.toml (hunk 1 above)
Comment (hypn0t1z) [reply data/ecosystems/e/easidao.toml]: I already updated, thank you

**Comment by hypn0t1z** [comment]
> we usually keep all repos including 404s, public repos that became private, forks, etc because we have ways to handle them

**Comment by hypn0t1z** [comment]
> Please don't remove the existing repos as we keep them for archival support.

Thank you for your feedback, I already updated

**Pull request closed without merging**